view/frontend/email/tesco_template.html

```diff
@@ -3,11 +3,10 @@
 <div>
     <p>
         {{trans "%thankyou" thankyou="Thank you for your order."}}</p>
-        {{trans "%message" message="This is your Tesco Lotus payment code. Please print it and bring it to any Tesco
-        Lotus store within the next 24 hours to finalise payment. After that time your order will be automatically
-        cancelled."}}<br/>
+        {{trans "%message" message="This is your Tesco Lotus payment code. Please print it and bring it to any Tesco Lotus store while 
+        the charge is still valid to finalize payment. After that time your order will be automatically cancelled."}}<br/>
```

Comment:
since we are changing `cancelled` to `canceled`. Maybe we need to change this too?